File: scala/problem006.scala
def a(n: Int): Int = {
  (1 to n).map(x => x * x).sum
}

def b(n: Int): Int = {
  val r = (1 to n).sum
  r * r
}

println(b(10) - a(10))
println(b(100) - a(100))

File: rust/problem006.rs
fn main() {
    let n = 100;
    let a: i32 = (1..=n).map(|i| i * i).sum();
    let b: i32 = (1..=n).sum::<i32>();
    let b = b * b;
    let ans = b - a;
    println!("{}", ans);
}
